samples/verifyIdToken-iap.js

```diff
@@ -49,14 +58,13 @@ async function main(
 
   // Print out the info contained in the IAP ID token
   console.log(ticket);
-
```

Comment:
since top level `await` is not yet supported in Node.js, we should pull the await bits into an `async` helper function, this is the documented format:

```
function main(arg1 = 'default_value_one', arg2 = 1234) {
  // [START your_region_tag]
  /**
   * TODO(developer): Uncomment these variables before running the sample.
   */
  // const arg1 = 'default_value_one';
  // const arg2 = 1234;

  // Imports the Google Cloud Some API library
  const {SomeApiClient} = require('@google-cloud/some-api');

  // Instantiates a client
  const someApiClient = new SomeApiClient();

  async function doSomething() {
    ...
  }

  doSomething();
  // [END your_region_tag]
}

main(...process.argv.slice(2));
```

Reply:
Ok I think I got it...